lang/src/main/kotlin/org/partiql/lang/eval/ExprValueType.kt

```diff
@@ -31,21 +31,22 @@ enum class ExprValueType(
     val isText: Boolean = false,
     val isLob: Boolean = false,
     val isSequence: Boolean = false,
-    val isRangedFrom: Boolean = false
+    val isRangedFrom: Boolean = false,
+    val isScalar: Boolean = false
 ) {
     MISSING(isUnknown = true),
     NULL(isUnknown = true),
-    BOOL,
-    INT(isNumber = true),
-    FLOAT(isNumber = true),
-    DECIMAL(isNumber = true),
-    DATE,
-    TIMESTAMP,
-    TIME,
-    SYMBOL(isText = true),
-    STRING(isText = true),
-    CLOB(isLob = true),
-    BLOB(isLob = true),
+    BOOL(isScalar = true),
+    INT(isNumber = true, isScalar = true),
+    FLOAT(isNumber = true, isScalar = true),
+    DECIMAL(isNumber = true, isScalar = true),
+    DATE(isScalar = true),
+    TIMESTAMP(isScalar = true),
+    TIME(isScalar = true),
+    SYMBOL(isText = true, isScalar = true),
+    STRING(isText = true, isScalar = true),
+    CLOB(isLob = true, isScalar = true),
+    BLOB(isLob = true, isScalar = true),
```

Comment:
The change here is for the change in `BagOp.kt`. See the comment there for more infor.